tests/postman/jsoverrides/admin_invites_code_POST.js

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env node
```

Comment:
Are all these files like this just placeholders and don't do anything yet? To be filled in later?

Reply:
these are not place holders they're important. they're manual overrides for the autogen test.